pkg/converter/convert_unix.go

```diff
@@ -304,7 +306,8 @@ func Pack(ctx context.Context, dest io.Writer, opt PackOption) (io.WriteCloser,
 			}
 		}()
 
-		if _, err := io.Copy(dest, blobFifo); err != nil {
+		buffer := make([]byte, defaultBufferSize)
```

Comment:
Maybe we can use [sync.Pool](https://github.com/containerd/containerd/blob/765351ac4def77c5c18e808ede3e71c99266fc6f/content/local/store.go#L514) to avoid allocating new buffer repeatedly, and do you have any performance testing, how about 1MB buffer?